examples/default.yaml

```diff
@@ -126,8 +126,8 @@ mounts:
 # 🟢 Builtin default: null
 mountTypesUnsupported: null
 
-# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (EXPERIMENTAL until Lima v1.0, from QEMU’s virtio-9p-pci, aka virtfs),
-# or "virtiofs" (EXPERIMENTAL, needs `vmType: vz`; will graduate from experimental in Lima v1.0)
+# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (QEMU’s virtio-9p-pci, aka virtfs),
+# or "virtiofs".
```

Comment:
Don't we want to keep the comment that `virtiofs` needs `vmType: vz`?

Reply:
No, on Linux

Reply:
```suggestion
# or "virtiofs" (needs `vmType: vz` on macOS).
```

Reply:
Added "needs vz on macOS"